### PR TITLE
updated dead link and added an example gif

### PR DESCRIPTION
**Pull request by KirtanSoni**
replaced CONTRIBUTING.md#create-a-pull-request with the current one.

## What type of PR is this? (check all applicable)

- [x] Documentation Update

## Related Tickets & Documents

- Related Issue # [Minor bugs in PR Template #17731](https://github.com/forem/forem/issues/17731)
- Closes #

## QA Instructions, Screenshots, Recordings

image
Changed Contributing guide link to a working one 

image
Added a Example Gif to promote that section.

## Added/updated tests?

- [x] No, and this is why: _Changes to Template documentation_

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -90,4 +90,6 @@ will share this change with the people who need to know about it._
 
 ## [optional] What gif best describes this PR or how it makes you feel?
 
-![alt_text](gif_link)
+<!-- ![gif-name](link)  or -->
+<img src="https://media.giphy.com/media/2KAGlmkPywhZS/giphy.gif" width="120" height="auto"/>
```
Comment (Ridhwana) [inline .github/PULL_REQUEST_TEMPLATE.md:94]: @KirtanSoni I think we may want the dead link here so that it makes it easy for folks to just copy and paste there gif link between the brackets

Comment (Ridhwana) [inline .github/PULL_REQUEST_TEMPLATE.md:94]: @KirtanSoni I think we may want the dead link here so that it makes it easy for folks to just copy and paste there gif link between the brackets

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -6,8 +6,8 @@
      commits if your PR already received reviews or comments.
 
      Before submitting a Pull Request, please ensure you've done the following:
-     - 📖 Read the Forem Contributing Guide: https://github.com/forem/forem/blob/main/CONTRIBUTING.md#create-a-pull-request.
-     - 📖 Read the Forem Code of Conduct: https://github.com/forem/forem/blob/main/CODE_OF_CONDUCT.md.
+     - 📖 Read the Forem Contributing Guide: https://developers.forem.com/contributing-guide/forem#create-a-pull-request
```
Comment (Ridhwana) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Nice one 🎉 thanks for catching this!

**Review by Ridhwana — COMMENTED** [review]
Hi @KirtanSoni 👋🏽 thanks for this ✨, I've left a small comment on the gif update.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (juliannatetreault) [reply .github/PULL_REQUEST_TEMPLATE.md:94]: Agreed, @Ridhwana. I think that once this change is reverted, this PR looks good to go, @KirtanSoni!

**Review by juliannatetreault — COMMENTED** [review]
This looks great, @KirtanSoni! I left one comment, that once addressed, should ready this PR for approval. 🌟

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -90,4 +90,6 @@ will share this change with the people who need to know about it._
 
 ## [optional] What gif best describes this PR or how it makes you feel?
 
-![alt_text](gif_link)
+<!-- ![gif-name](link)  or -->
+<!-- <img src="gif-link" width="120" height="auto"/> -->
```
Comment (Ridhwana) [inline .github/PULL_REQUEST_TEMPLATE.md:94]: @KirtanSoni do you think perhaps we should leave it as `alt_text`? The reason being is that if folks want to add a gif, then they don't need to uncomment the comments that you've added but can just replace `gif_link` with the actual link.

**Review by Ridhwana — COMMENTED** [review]
Thanks for reverting that @KirtanSoni ✨ I just have one more comment on the gif updates.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (juliannatetreault) [inline .github/PULL_REQUEST_TEMPLATE.md:94]: Quick suggestion: I would prefer that we fully revert this change and leave this as it was prior to changes (as shown below)--thoughts, @Ridhwana?
```suggestion
![alt_text](gif_link)
```

**Review by juliannatetreault — COMMENTED** [review]
Thank you so much for addressing our feedback, @KirtanSoni! I left one more, non-blocking comment and asked for input on it. 😊

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (juliannatetreault) [reply .github/PULL_REQUEST_TEMPLATE.md:94]: Ah, I just posed the same question and asked for your input, @Ridhwana! 😄

**Review by Ridhwana — APPROVED** [review]
Thank you @KirtanSoni 🎉This looks great
